tox.ini

```diff
@@ -313,49 +271,38 @@ commands_pre =
 
   jinja2: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-jinja2[test]
 
-  aiohttp-client: pip install {toxinidir}/opentelemetry-sdk {toxinidir}/instrumentation/opentelemetry-instrumentation-aiohttp-client
+  aiohttp-client: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-aiohttp-client
 
   aiopg: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-dbapi pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-aiopg[test]
 
   jaeger: pip install {toxinidir}/exporter/opentelemetry-exporter-jaeger
 
-  opentracing-shim: pip install {toxinidir}/opentelemetry-sdk
   opentracing-shim: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-opentracing-shim
 
-  datadog: pip install {toxinidir}/opentelemetry-sdk {toxinidir}/exporter/opentelemetry-exporter-datadog
+  datadog: pip install {toxinidir}/exporter/opentelemetry-exporter-datadog
 
   zipkin: pip install {toxinidir}/exporter/opentelemetry-exporter-zipkin
 
   sqlalchemy: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-sqlalchemy
 
   system-metrics: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-system-metrics[test]
 
-  elasticsearch{2,5,6,7}: pip install {toxinidir}/opentelemetry-instrumentation {toxinidir}/instrumentation/opentelemetry-instrumentation-elasticsearch[test]
+  elasticsearch{2,5,6,7}: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-elasticsearch[test]
 
 ; In order to get a healthy coverage report,
 ; we have to install packages in editable mode.
   coverage: python {toxinidir}/scripts/eachdist.py install --editable
 
-; Using file:// here because otherwise tox invokes just "pip install
-; opentelemetry-api", leading to an error
-  mypyinstalled: pip install file://{toxinidir}/opentelemetry-api/
-
 commands =
   test: pytest {posargs}
   coverage: {toxinidir}/scripts/coverage.sh
 
-  mypy: mypy --namespace-packages opentelemetry-api/src/opentelemetry/
-; For test code, we don't want to enforce the full mypy strictness
-  mypy: mypy --namespace-packages --config-file=mypy-relaxed.ini opentelemetry-api/tests/
-
-; Test that mypy can pick up typeinfo from an installed package (otherwise,
-; implicit Any due to unfollowed import would result).
-  mypyinstalled: mypy --namespace-packages opentelemetry-api/tests/mypysmoke.py --strict
-
 [testenv:lint]
```

Comment:
Even with these changes, the lint command fails because the code in the Core repo and the `skiptargetinstall-reference` code (the code that was originally donated to the project) has conflicting pylist issues. We have to either add these to an ignore which may require [a monkey patch due to an unresolved bug](https://github.com/PyCQA/pylint/issues/2686) or override the requirement and merge it in.